src/battle_controller_player.c

```diff
@@ -158,6 +162,8 @@ static void (*const sPlayerBufferCommands[CONTROLLER_CMDS_COUNT])(u32 battler) =
     [CONTROLLER_TERMINATOR_NOP]           = BtlController_TerminatorNop
 };
 
+static EWRAM_DATA bool8 sDescriptionSubmenu = 0;
+
 static EWRAM_DATA bool8 sAckBallUseBtn = FALSE;
 static EWRAM_DATA bool8 sBallSwapped = FALSE;
```

Comment:
The battle debug menu uses some hack to avoid using an `EWARM`. Not for this PR but we should explore it at some point. Maybe we will be able to remove those. I'll open an issue once it is merged.